### PR TITLE
Bump capiVersion to 17.2

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Bump capiVersion to 17.2. (Mostly to receive the Interactive Atom `placeholderUrl` from here: https://github.com/guardian/content-api-scala-client/commit/a558bb8261e46f47d7b617b03bcd039ef1c8772c ). This also performs the cleaning of few tests that needed to be updated. 

nb (1): I could not recover test: "Video as main element should act as main media". Willing to reconstruct it later.

nb (2): HostedTrailsTest.scala is no longer compiling as it is and is not testing much.

### CODE REVIEW

File: article/test/ArticleFeatureTest.scala [hunk 1]
```diff
@@ -406,31 +371,6 @@ import scala.collection.JavaConverters._
...
-          include(
-            "img/static/sys-images/Guardian/Pix/audio/video/2014/5/16/1400240928538/Nigel-Farage-LBC-debate-i-014.jpg?width=640&quality=85&auto=format&fit=max&s=",
-          )
-
-        And("video meta uploadDate should be set")
-        $("[itemprop='associatedMedia video'] [itemprop=uploadDate]").attribute("content") should be(
-          "2014-05-16T16:09:34.000+01:00",
-        )
-      }
-    }
```
Comment (jfsoul) [inline article/test/ArticleFeatureTest.scala:432]: How come we needed to remove tests here and elsewhere?

**Review by jfsoul — COMMENTED** [review]
Just the question about the tests being removed really.

File: article/test/ArticleFeatureTest.scala (hunk 1 above)
Comment (shtukas) [reply article/test/ArticleFeatureTest.scala:432]: Now that you mention it, let me try and re-introduce just this one.

**Comment by mxdvl** [comment]
> nb (2): HostedTrailsTest.scala is no longer compiling as it is and is not testing much.

It seems like this falls within the Commercial Dev team’s remit. Do you want me to confirm with the team that we can get rid of this test?

**Comment by shtukas** [comment]
@mxdvl Yes, Yes, Yes! Thank You! I removed it because it's not compiling anymore, and I could not fix it.

File: article/test/ArticleFeatureTest.scala (hunk 1 above)
Comment (shtukas) [reply article/test/ArticleFeatureTest.scala]: @jfsoul This commit ( https://github.com/guardian/frontend/pull/23056/commits/51e16fe36df9cadc5a673e2db2229933d0452413 ), the verbatim origin test, breaks the build by failing. I could not figure out why 🤔 I am going to revert it

**Review by mxdvl — APPROVED** [review]
Commercial is happy to forgo the HostedTrails tests for now!

**Comment by shtukas** [comment]
@mxdvl Awesome! 👍

**Pull request merged**